pkgs/applications/office/tagainijisho/default.nix

```diff
@@ -1,16 +1,19 @@
-{ lib, stdenv, fetchurl, qt4, cmake, sqlite }:
-
-stdenv.mkDerivation rec {
+{ lib, mkDerivation, fetchzip, qtbase, qttools, cmake, sqlite }:
+mkDerivation rec {
   pname = "tagainijisho";
-  version = "1.0.3";
+  version = "1.2.0";
 
-  src = fetchurl {
+  src = fetchzip {
```

Comment:
`fetchurl` should be fine, right?

Reply:
both should work. idk if one or the other is/should be preferred. I like `fetchzip`, because it gives you a folder for `.src`, but I guess that's just personal preference.